src/lib/logger.ts

```diff
@@ -0,0 +1,18 @@
+import { Config } from "react-native-config";
+
+export const debugLogger = (message: string, data?: any) => {
+  if (!!Config.IS_DEV) {
+    if (data) {
+      console.log(message)
+      console.log(data)
+    } else {
+      console.log(message)
+    }
+  }
+}
+
+export const errorLogger = (fileName: string, functionName: string, error: any) => {
+  console.error(fileName)
```

Comment:
Going through the PR it looks like you're only passing 2 strings to this function when you call it. Maybe you can get rid of the 3 arguments here and just have `(info:string, error:any)`.